lithops/serverless/backends/k8s/entry_point.py

```diff
@@ -122,6 +121,130 @@ def run_job(payload):
 
     logger.info("Finishing kubernetes execution")
 
+def run_job_k8s_rabbitmq(payload, job_index, running_jobs):
+    logger.info(f"Lithops v{__version__} - Starting kubernetes execution")
+
+    act_id = str(uuid.uuid4()).replace('-', '')[:12]
+    os.environ['__LITHOPS_ACTIVATION_ID'] = act_id
+    os.environ['__LITHOPS_BACKEND'] = 'k8s_rabbitmq'
+    
+    payload['call_ids']  = [payload['call_ids'][job_index]]
+    payload['data_byte_ranges'] = [payload['data_byte_ranges'][job_index]]
+
+    function_handler(payload)
+    running_jobs.value -= 1
+
+    logger.info("Finishing kubernetes execution")
+
+# Function to calculate the number of executions of this pod
+def calculate_executions(total_cpus_cluster, pod_cpus, range_ids_pod, total_functions):
+    base_executions = total_functions // total_cpus_cluster
+    remaining_executions = total_functions % total_cpus_cluster
+
+    # Calculate the number of executions based on the pod CPUs and the number of executions
+    pod_executions = pod_cpus * base_executions
+    
+    if range_ids_pod[0] <= remaining_executions <= range_ids_pod[1]:
+        remaining_executions = remaining_executions - range_ids_pod[0]
+        pod_executions = pod_executions + remaining_executions
+        return pod_executions, base_executions
+    
+    if remaining_executions > range_ids_pod[0]:
+        pod_executions = pod_executions + pod_cpus
+
+    return pod_executions, base_executions
+
+# Callback to receive the payload and run the jobs
+def callback_run_jobs(ch, method, properties, body):
+    global range_start, range_end, total_cpus_cluster
+    payload = json.loads(body)
+
+    total_calls = payload['total_calls']
+    requested_cpus = 0
+
+    logger.info(f"Call from lithops received.")
+
+    try:
+        # Calculate the number of executions of this pod
+        if total_calls > range_start:
+            if range_end > total_calls - 1:
+                requested_cpus = total_calls - range_start
+            else:
+                requested_cpus = range_end - range_start + 1
+        else:  # No more executions to do to this pod
+            return
+
+        pod_cpus = range_end - range_start + 1
+        total_executions, bases_executions = calculate_executions(total_cpus_cluster, pod_cpus, [range_start, range_end], total_calls)
```

Comment:
Can you explain what is `range_start`, `range_end`,  `total_executions` and `bases_executions`?